docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+<!--
+---
+linkTitle: "Chains:v2"
+weight: 10
+---
+-->
+
+# Build Type
+
+This buildType is maintained by [TektonCD/Chains](https://github.com/tektoncd/chains) and describes a build executed by [TektonCD/Pipelines](https://github.com/tektoncd/pipeline).
+
+
+# Description
+
+```
+"buildType": "https://tekton.dev/predicate/chains/slsa/v2"
```

Comment:
similarly, I'd prefer to name the build type v1.0 to avoid confusion.

slsa/v1. slsa/v2alpha2 are just internal alias of the formatter name. Other than config map, I think we should avoid use those aliases for user-facing doc i.e. this build type doc.

Reply:
Just something to note, here are the buildTypes I used for https://github.com/tektoncd/chains/pull/895. Whatever is chosen, we should sync with that change.

Reply:
agreed. We can make the user facing changes in a follow-up since we map this to our internal repo structure.